README.md

```diff
@@ -291,6 +291,7 @@ quality tools, including:
 - [pep8-naming](https://pypi.org/project/pep8-naming/)
 - [pydocstyle](https://pypi.org/project/pydocstyle/)
 - [pygrep-hooks](https://github.com/pre-commit/pygrep-hooks)
+- [pylint-airflow](https://pypi.org/project/pylint-airflow/)
```

Comment:
I decided to link to the Pylint plugin, since this list refers to tools we've reimplemented, and it felt more appropriate than "Airflow" -- wdyt?

Reply:
yeah this makes sense to me! hopefully we get to a point where we're linting for more than just pylint-airflow though 🙂